applications/FluidDynamicsApplication/symbolic_generation/two_fluid_navier_stokes/two_fluid_navier_stokes_template.cpp

```diff
@@ -53,7 +53,7 @@ Element::Pointer TwoFluidNavierStokes<TElementData>::Create(
     NodesArrayType const &ThisNodes,
```

Comment:
Just a reminder to add the code review suggestions to this template.

Reply:
Done.